x-pack/plugins/alerting/server/routes/lib/rewrite_request_case.ts

```diff
@@ -24,15 +24,15 @@ type RenameAlertToRule<K extends string> = K extends `alertTypeId`
 
 export type AsApiContract<
   T,
-  ComplexPropertyKeys = `actions` | `executionStatus` | 'lastRun',
+  ComplexPropertyKeys = `actions` | `executionStatus` | 'lastRun' | `frequency`,
```

Comment:
```suggestion
  ComplexPropertyKeys = 'actions' | 'executionStatus' | 'lastRun' | 'frequency',
```

Reply:
Huh yeah I was wondering why these were all backticks and ESLint didn't have anything to say about it